### PR TITLE
Remove the deprecated oc-create-kubeconfig cmd in local-exe

### PR DESCRIPTION
**Pull request by hongkailiu**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -294,7 +294,9 @@ validate-checkconfig:
 .PHONY: validate-checkconfig
 
 $(TMPDIR)/.ci-operator-kubeconfig:
-	oc --context $(CLUSTER) --as system:admin --namespace ci serviceaccounts create-kubeconfig ci-operator > $(TMPDIR)/.ci-operator-kubeconfig
+	oc --context $(CLUSTER) -n test-credentials extract secret/ci-operator --to=$(TMPDIR)
+	mv $(TMPDIR)/kubeconfig "$@"
```
[the remark this answers is not in the document]
Comment (hongkailiu) [reply Makefile:298]: There are two files. Currently, we use `--keys kubeconfig`.
Very soon, we are going to need both of them.

[the remark this answers is not in the document]
Comment (bbguimaraes) [reply Makefile:298]: You'd still in that case have one target per file and extract them individually, wouldn't you?

**Pull request merged**